core/client/src/main/java/alluxio/client/file/FileOutStream.java

```diff
@@ -116,7 +120,8 @@ public FileOutStream(AlluxioURI path, OutStreamOptions options) throws IOExcepti
         String tmpPath = PathUtils.temporaryFileName(mNonce, mUfsPath);
         UnderFileSystem ufs = UnderFileSystem.get(tmpPath, ClientContext.getConf());
         // TODO(jiri): Implement collection of temporary files left behind by dead clients.
-        mUnderStorageOutputStream = ufs.create(tmpPath, (int) mBlockSize);
+        CreateOptions createOptions = new CreateOptions().setPermission(options.getPermission());
```

Comment:
I kinda agree with @jsimsa here as I understand the purpose of using `xxxOptions` instead of `xxxTOptions` directly is to hide the specific way of sending data over wire, and easing the development is probably the final goal